sql/migrations.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE IF NOT EXISTS pg2kafka.outbound_event_queue (
   table_name    varchar(255) NOT NULL,
   statement     varchar(20) NOT NULL,
   data          jsonb NOT NULL,
+  previous_data jsonb NOT NULL,
```

Comment:
This will not work for existing installations, will it? I think we need to think of a way to apply this to existing systems as well

Reply:
Also want to give the naming here some more thought